### ISSUE TITLE
LanguagesNav should be keyboard navigable

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#176]
The LanguagesNav cannot be navigated using the keyboard alone.

This may require updating the CSS and possibly the JS to take into account the new HTML structure (or not).
